net-nntp/sabnzbd/sabnzbd-4.1.0.ebuild

```diff
@@ -0,0 +1,168 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+PYTHON_COMPAT=( python3_{9..11} )
+PYTHON_REQ_USE="sqlite"
+
+inherit optfeature python-single-r1 systemd
+
+MY_PV="${PV/_alpha/Alpha}"
+MY_PV="${MY_PV/_beta/Beta}"
+MY_PV="${MY_PV/_rc/RC}"
+
+MY_P="${PN/sab/SAB}-${MY_PV}"
+
+DESCRIPTION="Binary newsgrabber with web-interface"
+HOMEPAGE="https://sabnzbd.org/"
+SRC_URI="https://github.com/sabnzbd/sabnzbd/releases/download/${MY_PV}/${MY_P}-src.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+# Sabnzbd is GPL-2 but bundles software with the following licenses.
+LICENSE="GPL-2 BSD LGPL-2 MIT BSD-1"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="
+	acct-user/sabnzbd
+	acct-group/sabnzbd
+	${PYTHON_DEPS}
+	$(python_gen_cond_dep '
+		dev-python/chardet[${PYTHON_USEDEP}]
+		dev-python/cheetah3[${PYTHON_USEDEP}]
+		dev-python/cherrypy[${PYTHON_USEDEP}]
+		dev-python/configobj[${PYTHON_USEDEP}]
+		dev-python/cryptography[${PYTHON_USEDEP}]
+		>=dev-python/feedparser-6.0.10[${PYTHON_USEDEP}]
+		>=dev-python/guessit-3.7.1[${PYTHON_USEDEP}]
+		dev-python/notify2[${PYTHON_USEDEP}]
+		dev-python/portend[${PYTHON_USEDEP}]
+		dev-python/puremagic[${PYTHON_USEDEP}]
+		~dev-python/sabctools-7.1.2[${PYTHON_USEDEP}]
+	')
+		test? ( $(python_gen_cond_dep '
+				dev-python/tavalidate[${PYTHON_USEDEP}]
+				>=dev-python/tavern-2[${PYTHON_USEDEP}]
+			')
+		)
+"
+RDEPEND="
+	${DEPEND}
```

Comment:
This copies test dependencies into RDEPEND which is wrong.

Also, it seems that test dependencies are now randomly split between `DEPEND` and `BDEPEND`.

Reply:
Good catch, I've moved all the test dependencies to BDEPEND.